source/Calamari.AzureCloudService.Tests/DeployAzureCloudServiceCommandFixture.cs

```diff
@@ -169,8 +172,8 @@ public async Task Deploy_Ensure_Tools_Are_Configured()
         {
             var serviceName = $"{nameof(DeployAzureCloudServiceCommandFixture)}-{Guid.NewGuid().ToString("N").Substring(0, 12)}";
             var deploymentSlot = DeploymentSlot.Staging;
-
-            using var client = new ComputeManagementClient(subscriptionCloudCredentials);
+            var httpClientProxy = new AuthHttpClientFactory().GetHttpClient();
```

Comment:
It seems there is some inconsistency in how we get the client:
- `new AuthHttpClientFactory().GetHttpClient()`
- `AuthHttpClientFactory.ProxyClientHandler()`

How would someone know which to use?

Reply:
The `ComputeManagementClient` only takes a httpClient while others take a httpClientHandler. the httpClientHandler looks to be the way moving forward.

Reply:
👍